reference/api/keys.md

```diff
@@ -1,30 +1,264 @@
+| documents.add    | Provides access to `POST` and `PUT` on `/indexes/:authorizedIndexes/documents` routes. |
+| documents.get    | Provides access to `GET` methods on `/indexes/:authorizedIndexes/documents` and `/indexes/:authorizedIndexes/documents/:documentId` routes. |
+| documents.delete | Provides access to `DELETE` method on `/indexes/:authorizedIndexes/documents/:documentId`, `indexes/:authorizedIndexes/documents/:documentId` and `POST` method on `/indexes/:authorizedIndexes/documents/delete-batch` routes. |
+| indexes.add      | Provides access to `POST` `/indexes`. **⚠️ The newly created index is automatically added to the permissions for the API key making the call, as well as all others keys having `[*]` as a value for `indexes`**. |
+| indexes.get      | Provides access to `GET` `/indexes` and `/indexes/:authorizedIndexes`. **⚠️Non-authorized `indexes` are omitted from the response on `/indexes`**. |
+| indexes.update   | Provides access to `PUT` `/indexes/:authorizedIndexes`. |
+| indexes.delete   | Provides access to `DELETE` `/indexes/:authorizedIndexes`. |
+| tasks.get        | Provides access to `GET` `/tasks`. **⚠️Non-authorized `indexes` are omitted from the response on `/tasks`**. Also add access to `GET` `/indexes/:authorizedIndexes/tasks` routes. |
+| settings.get     | Provides access to `GET` `/indexes/:authorizedIndexes/settings` and `/indexes/:authorizedIndexes/settings/*` routes. |
+| settings.update  | Provides access to `POST / DELETE` `/indexes/:authorizedIndexes/settings` and `/indexes/:authorizedIndexes/settings/*` routes. |
+| stats.get        | Provides access to `GET` `/stats/`. **⚠️Non-authorized `indexes` are omitted from the response on `/stats`**. Also add access to `GET` `/indexes/:authorizedIndexes/stats`. |
+| dumps.create     | Provides access to `POST` `/dumps` route. **As dumps are not scoped by indexes, a restriction on `indexes` does not affect this action.** |
+| dumps.get        | Provides access to `GET` `/dumps/:dumpUid` route. **As dumps are not scoped by indexes, a restriction on `indexes` does not affect this action.** |
+| version          | Provides access to `GET` `/version` route.
+
+#### `indexes`
+
+**Type:** array
+**Modifiable:** yes
+**Mandatory:** yes
+
+A list of indexes permitted for the key. `["*"]` for all indexes.
+
+#### `expiresAt`
+
+**Type:** string
+**Modifiable:** yes
+**Mandatory:** yes
+
+Expiration date and time of the key represented in **ISO-8601** format. `null` if the key never expires.
+
+#### `createdAt`
+
+**Type:** string
+**Modifiable:** no
+
+Date and time the key was created, represented in **ISO-8601** format.
+
+#### `updatedAt`
+
+**Type:** string
+**Modifiable:** no
+
+Date and time the key was last updated, represented in **ISO-8601** format.
+
+## Get one key
+
+<RouteHighlighter method="GET" route="/keys/:key"/>
+
+Get information on the specified key.
```

Comment:
Should we say something about what happens when you use `GET keys/:key` with a deleted/expired key?